pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ds-caselaw-marklogic-api-client"
-version = "28.0.0"
+version = "29.0.0-alpha.2"
```

Comment:
I think this is leftover from testing, we can remove this and then do a proper version bump later when SQIDs are in.

Reply:
Done.